ex1/ex1.c

```diff
@@ -9,7 +9,24 @@
 int main(void)
 {
     // Your code here
-    int x = 
+    
```

Comment:
Looks good to me. The only thing it might need is a condition for when the fork fails("letsfork" is less than 0). Great Job Overall.